src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -114,11 +115,13 @@ public bool IsKnownDevice
         public Command LogInCommand { get; }
         public Command TogglePasswordCommand { get; }
         public ICommand MoreCommand { get; internal set; }
+        public AsyncCommand LogInWithDeviceCommand { get; }
```

Comment:
If possible, just use `ICommand` for the definition of the command property unless you need some access to something specific on the `AsyncCommand`. This way, if then you want to change the command implementation you only have to do it once, like when we migrate to MAUI